requirements.txt

```diff
@@ -1,6 +1,7 @@
 anyconfig>=0.9.7, <1.0
 click<8.0
 cookiecutter>=1.6.0, <2.0
+dask>=2.4.0
```

Comment:
Please move this in `setup.py` `extras_require` using a new `dask` key. Thanks :)